### PR TITLE
completed the student table

### LINKED ISSUE #16: Create Student Table
In this Issue you will create a React component for displaying data from the /api/students/all endpoint.

You can populate that endpoint by using the /api/students/upload/egrades endpoint on swagger with data from /docs/examples/egrades.csv

The table should show the available fields from the  /api/students/all  endpoint.

# Acceptance Criteria

- [x] The React component described above exists
- [x] It has a storybook entry
- [x] It has test coverage

# Discussion

This will eventually be used on the Course Show Page #17

### PR DESCRIPTION
**Pull request by GuBinhti**
Created the Table for student as well as the story, fixtures, and tests

close #16 

Edit: 
Expanded Student Table to match all the columns specified in the Kanban Board Item/Backend URL endpoint:
image
image

### CODE REVIEW

File: frontend/src/tests/components/Student/StudentTable.test.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import StudentTable from "main/components/Student/StudentTable"
+import { currentUserFixtures } from "fixtures/currentUserFixtures";
+// import { fireEvent, render, waitFor, screen } from "@testing-library/react";
```
Comment (pconrad) [inline frontend/src/tests/components/Student/StudentTable.test.js]: Commented out code.

File: frontend/src/tests/components/Student/StudentTable.test.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      const header = screen.getByTestId(`${testId}-cell-row-0-col-${field}`);
+      expect(header).toBeInTheDocument();
+    });
+
+    expect(screen.getByTestId(`${testId}-cell-row-0-col-id`)).toHaveTextContent("1");
+    expect(screen.getByTestId(`${testId}-cell-row-1-col-id`)).toHaveTextContent("2");
+
+    // const deleteButton = screen.getByTestId(`${testId}-cell-row-0-col-Delete-button`);
+    // expect(deleteButton).toBeInTheDocument();
+    // expect(deleteButton).toHaveClass("btn-danger");
```
Comment (pconrad) [inline frontend/src/tests/components/Student/StudentTable.test.js]: commented out code

**Review by pconrad — CHANGES_REQUESTED** [review]
commented out code.  otherwise LGTM.

**Comment by jason-rodrigues** [comment]
Edit: 
Expanded Student Table to match all the columns specified in the Kanban Board Item/Backend URL endpoint:
image
image

[Storybook](https://ucsb-cs156-s24.github.io/proj-organic-s24-5pm-4/prs/39/storybook/?path=%2Fstory%2Fcomponents-student-studenttable--empty)

**Review by SR33G33 — APPROVED** [review]
LGTM 😎 !! I can see the table on the gh-pages storybook

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**